### ISSUE TITLE
Toolbar position

### ISSUE THREAD

**Issue by LeonidPelyhivskyi** [michelson/dante2#129]
The toolbar has wrong position and size on small resolutions
<img width="438" alt="screen shot 2018-09-27 at 16 00 37" src="https://user-images.githubusercontent.com/6578012/46147608-9d891d00-c26e-11e8-8b2b-c453f9f561d4.png">
Do you have any fixes for this?

**Comment by michelson** [comment]
Hi @LeonidPelyhivskyi , position looks good to me, maybe it's not quite pixel perfect yet. 
The size could be handled by hiding some elements on media query. I guess that will depend on each implementation.

**Comment by LeonidPelyhivskyi** [comment]
Hi @michelson, the position looks good, you right. I meant toolbar size. I am not sure that is a good idea to hide elements. What I should do if I need all of them? I think it much better if the editor will control the size of the toolbar or by default will be in two line if it can't be rendered in the viewport. 

A few more cases:

<img width="385" alt="screen shot 2018-09-27 at 20 37 14" src="https://user-images.githubusercontent.com/6578012/46163924-421e5580-c295-11e8-8b88-3248c61cb494.png">


<img width="361" alt="screen shot 2018-09-27 at 20 37 42" src="https://user-images.githubusercontent.com/6578012/46163925-421e5580-c295-11e8-931e-86e0efffcc26.png">

**Comment by michelson** [comment]
Hi @LeonidPelyhivskyi,  maybe it would make more sense to set position of the toolbar as fixed (editor's top or bottom ) instead popover mode on mobile context. ¿what do you think about that?

**Comment by LeonidPelyhivskyi** [comment]
Yes, it makes sense. But it will not fix toolbar width. You should do more manipulation with toolbar elements to make it smaller. In any case, it will be much better than now.

**Comment by blanck-space** [comment]
I've noticed the same issue @LeonidPelyhivskyi however I was able to override with custom CSS rules to make it "perfect" for my implementation. I agree that Dante2 repo itself can have more modular CSS handling but until that gets addressed; just override or fork and change the CSS.

**Comment by michelson** [comment]
check version 0.5.0-rc13 👍 

## I've implemented a fixed version of the menu. for mobile will render as:

![image](https://user-images.githubusercontent.com/11976/52766819-267ad800-3007-11e9-81fe-fa1858a4da1c.png)


## for wide screen will  render as:

![image](https://user-images.githubusercontent.com/11976/52766830-3397c700-3007-11e9-9947-d3c49a1fc8ea.png)


> Also you can force the sticky behavior to be always fixed just setting the `sticky: true` on the toolbar block
